methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+
+def get_word(wordtype)
+  puts "Give me #{wordtype} "
```

Comment:
`print` would be better in my opinion.  That way it says `Give me an Adjective:` instead of `Give me an Adjective:` followed by a new line.